.circleci/config.yml

```diff
@@ -236,6 +236,7 @@ jobs:
       - run:
           name: "Build"
           command: |
+            dnf install -y libicu-devel 
```

Comment:
we should probably move it to the setup-* scripts and have it baked in the docker images- So that its easy to repro problems rather than trying to wonder why build isnt working.

Reply:
Ah yes agreed, this is from before the ICU PR :D